api/api/views/sandbox_survey_view.py

```diff
@@ -1,12 +1,51 @@
-
+from django.utils import timezone
 from rest_framework import permissions, generics
+from rest_framework.response import Response
+from api.models.sandbox_survey import SandboxSurvey
+from django.http import HttpResponseBadRequest
+import json
+
 
 class SandboxSurveyView(generics.GenericAPIView):
-    permission_classes = (permissions.IsAuthenticated,)
-
-    def get(self, request, name=None):
-        "hello"
-        
-    def put(self, request, pk, name=None):
-        "hello"
-    
+    # permission_classes = (permissions.IsAuthenticated,)
```

Comment:
Shouldn't be making it into master/main with this commented out.

Reply:
😨 Oops yep definitely forgot to re-enable that. Good catch!